types/three/three-core.d.ts

```diff
@@ -6643,10 +6643,9 @@ export namespace SceneUtils {
 }
 
 export namespace ShapeUtils {
-    export function area(contour: number[]): number;
-    export function triangulate(contour: number[], indices: boolean): number[];
-    export function triangulateShape(contour: number[], holes: any[]): number[];
-    export function isClockWise(pts: number[]): boolean;
+    export function area(contour: (Vector2 | Vector3)[]): number;
```

Comment:
Seems unlikely to be correct - can someone actually pass a *mix* of `Vector2` and `Vector3` in the same array into this function?

Reply:
Maybe I changed it to
```contour: Vector2[] | Vector3[]```
Would it be better?